modules/aws-load-balancer-operator-uninstall.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * networking/aws-load-balancer-operator.adoc
+
+:_content-type: PROCEDURE
+[id="aws-load-balancer-operator-uninstall_{context}"]
+= Uninstalling an AWS Load Balancer Operator
+To uninstall an AWS Load Balancer Operator (ALBO) and perform an overall clean up of the associated resources, perform the following procedure.
+
+.Procedure
+. Clean up the sample application by deleting the Load Balancers created and managed by the ALBO. For more information, reagrding deleting Load Balancers,  see link:https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-delete.html[Delete an Application Load Balancer].
+. Clean up the AWS VPC tags by removing the VPC tags that were added to the subnets for discovering subnets and for creating Application Load Balancers (ALBs). For more information, see link:https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-delete.html[Remove tags from resource].
```

Comment:
The link in this step goes to the same page as in the previous step about deleting an ALB. There's no info on that page about cleaning up tags.

Reply:
Good catch. Changed it, thanks.